scripts/set_gem_timing

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+NC_STRING="NC"
+SC_STRING="SC"
+# TODO:(josh) reflect on absolute paths
+IOC_PATH_NC="/reg/g/pcds/epics/ioc/fee/GasDetDAQ/R4.0.22"
+IOC_PATH_SC="/reg/g/pcds/epics/ioc/fee/GasDetDAQ/R4.0.22-NOTS"
+IOC_PATH=0
+
+# User input validation
+if [[ -n $1 ]]; then # if the user passed at least one arg
+  if [[ $1 != $NC_STRING ]] && [[ $1 != $SC_STRING ]]; then #validate argument is either SC or NC
+    echo "Error. Argument must be either: [NC or SC]"
+    exit 1
+  else
+    echo "User passed argument of: $1"
+  fi
+else
+  echo "Error. Must pass at least one argument"
+  exit 1
+fi
+
+if [[ -n $2 ]]; then
+  echo "Warning. Must pass strictly one argument, additional arg: $2 is ignored"
+  exit 1
+fi
+
+# check current operational state
+OP_MODE=$(caget KFE:CAM:TPR:02:MODE | awk '{print $2}')
+SET_MODE_CMD="caput KFE:CAM:TPR:02:MODE $1"
```

Comment:
What's the purpose of breaking this out as a string? I don't see anywhere else where this is used.